core/src/com/unciv/logic/city/CityExpansionManager.kt

```diff
@@ -33,11 +33,17 @@ class CityExpansionManager {
     // The second seems to be more based, so I'll go with that
     fun getCultureToNextTile(): Int {
         var cultureToNextTile = 6 * (max(0, tilesClaimed()) + 1.4813).pow(1.3)
+        for (unique in cityInfo.civInfo.getMatchingUniques("-[]% Culture cost of acquiring tiles []")) {
+            if (cityInfo.matchesFilter(unique.params[1]))
+                cultureToNextTile *= (1 - unique.params[0].toFloat() / 100)
```

Comment:
`100-`... instead of `1-`
Also you don't need the parentheses around the value

Reply:
I changed the format to more closely match the method used to calculate maintenance cost reduction. Previously I had used the method used to calculate unhappiness reduction. How does it look now?

Reply:
Ah, excuse me. I had previously had a 100f in there instead of just a straight 100, because I saw that in a different context and thought it would be necessary here, but no, a regular 100 will be just fine.